test/data/adult.tiny.with-schema.txt

```diff
@@ -1,5 +1,6 @@
 #@ TextLoader{
 #@   header+
+#@   sparse+
```

Comment:
What is sparse about this dataset?

Reply:
Removed.

---
In reply to: [259421045](https://github.com/dotnet/machinelearning/pull/2630#discussion_r259421045) [](ancestors = 259421045)